pkg/maps/ipcache/ipcache.go

```diff
@@ -85,5 +88,8 @@ var (
 )
 
 func init() {
-	bpf.OpenAfterMount(IPCache)
+	err := bpf.OpenAfterMount(IPCache)
+	if err != nil {
+		log.WithError(err).Error("unable to open map")
```

Comment:
Should we fatal? If not, what is the point of logging an error if the user can't do anything about it. the `moutFS()` will never be called again

Reply:
Our current understanding is that this fails on minikube early on. We don't know why. Maps which are opened on demand or from the datapath side will succeed. Making this Fatal will break Cilium on Minikube.